WooCommerce/Classes/ViewRelated/Editor/AztecEditorViewController.swift

```diff
@@ -328,7 +334,27 @@ extension AztecEditorViewController: UITextViewDelegate {
 }
 
 private extension AztecEditorViewController {
-    func showProductGeneratorBottomSheet() {
-        // TODO: 9465 - show bottom sheet
+    func showDescriptionGenerationBottomSheet() {
+        let presenter = BottomSheetPresenter(configure: { bottomSheet in
+            var sheet = bottomSheet
+            sheet.prefersEdgeAttachedInCompactHeight = true
+            sheet.largestUndimmedDetentIdentifier = .large
+            sheet.prefersGrabberVisible = true
+            sheet.detents = [.medium(), .large()]
+        })
+        bottomSheetPresenter = presenter
+
+        // TODO: 9465 - show product description generation in the bottom sheet
+        let controller = UIViewController()
+        controller.view.backgroundColor = .primaryButtonBackground
+
+        view.endEditing(true)
+        presenter.present(controller, from: self) { [weak self] in
+            self?.richTextView.becomeFirstResponder()
+        }
```

Comment:
Super nit: we should keep the param label here to make it easier to understand. The trailing closure doesn't clarify that it's for when the controller is dismissed.

Also, when dismissing the description view, this closure will attempt to make the `richTextView` the first responder again. I wonder if we can omit the closure then:


https://user-images.githubusercontent.com/5533851/234503528-3c99749b-9c41-4cdb-ba9b-8a49163d2057.mp4

Reply:
I couldn't reproduce this on a few iOS 16 simulators, what's the type of simulator/device you used? In any case, I made some changes to allow dismissing the bottom sheet programmatically with a different completion callback and please see if it's fixed https://github.com/woocommerce/woocommerce-ios/pull/9552/commits/1d6f0221b68e49efd482f10933dfb6f74450890b

Reply:
I was using simulator iOS 16.2 - but the new fix seems to have made it go away 👍